src/plone/restapi/services/site/get.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from plone.registry.interfaces import IRegistry
+from plone.restapi.services import Service
+from Products.CMFPlone.interfaces import IImagingSchema
+from Products.CMFPlone.interfaces import ISiteSchema
+from Products.CMFPlone.utils import getSiteLogo
+from zope.component import getMultiAdapter
+from zope.component import getUtility
+
+
+class SiteGet(Service):
+    def reply(self):
+        portal_state = getMultiAdapter(
+            (self.context, self.request), name="plone_portal_state"
+        )
+        registry = getUtility(IRegistry)
+        site_settings = registry.forInterface(ISiteSchema, prefix="plone", check=False)
+        image_settings = registry.forInterface(
+            IImagingSchema, prefix="plone", check=False
+        )
+        return {
+            "title": portal_state.portal_title(),
+            "logo": site_settings.site_logo and getSiteLogo() or None,
+            "robots_txt": site_settings.robots_txt,
+            "imaging_allowed_sizes": image_settings.allowed_sizes,
```

Comment:
@erral I am wondering if we shouldn't put a bit more effort into finding a naming convention for plone.registry entries here. In the future, we might want to expose more plone.registry entries via the site endpoint. Just thinking out loud here: what if we would add something like:

````
{
    "registry": {
        "plone.allowed_sizes": [
            "huge 1600:65536",
            ...
        ],
    }
}
````

This way we could explicitly add more plone.registry entries in the future and integrators could override the endpoint if they want to expose certain plone.registry entries on project level.

In the mid/far future, we could add functionality to plone.registry to put permissions on registry entries and automatically expose registry entries here (though, in that case we might want to incorporate this in the registry endpoint itself).


Would love to hear @sneridagh opinion on this, since exposing plone.registry entries via zope2.View was on our todo list since quite a while.

Reply:
No problem with renaming the attributes to match registry names, I find it OK.

Regarding registry options, we currently have a registry endpoint but [it requires `cmf.ManagePortal`](https://github.com/plone/plone.restapi/blob/master/src/plone/restapi/services/registry/configure.zcml#L10)

It is hard to decide which registry entries are public and thus can be publicly exposed. This way of adding these to the `@site` endpoint is a way to expose some of then on a need-basis.

In some of our projects where we save some data in the registry, we have overriden the @registry endpoint to be able to obtain given records by anonymous users, in that case we do that checking the registry name prefix. I don't know if this approach could be feasible here because most of Plone's records are prefixed with `plone` but not with `plone.imagin.allowed_size` or whatever.

Reply:
I don't think we need to decide what registry entries should be public. This is something an integrator can decide. I just want us to use a naming convention that allows integrators to stay consistent when they want to expose a particular plone.registry entries via the @site endpoint as well. This might be more convenient than overriding the existing registry endpoint. Both are valid approaches and, as always, it depends on the use case which approach is better in a certain situation.

Reply:
I have reworked the exposed property names to match registry record names